templates/build/index.html

```diff
@@ -164,7 +164,7 @@ <h2><span class="u-number-circle">2</span> What version of Ubuntu do you want?</
     <div class="row  js-selection-container">
       <div class="col-3">
         <button class="p-button is-wide js-selection" data-supports="raspberrypi2 raspberrypi3">
-          <span class="js-name">Classic 16.04</span>
+          <span class="js-name">16.04 LTS</span>
```

Comment:
The slug we use in the dictionary is hard-tied to the text we display to the user? That's a bad system.

Reply:
Done.